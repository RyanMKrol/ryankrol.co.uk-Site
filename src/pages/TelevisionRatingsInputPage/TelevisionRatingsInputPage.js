import React from 'react';

import { TelevisionRatingsForm } from '../../components';

import styles from './TelevisionRatingsInputPage.module.css';

/**
 * Page introducing the site
 *
 * @returns {string} JSX Content
 */
function TelevisionRatingsInputPage() {
  return (
    <div className={styles.container}>
      <h1 className="light">Rate a TV show!</h1>
      <TelevisionRatingsForm />
    </div>
  );
}

export default TelevisionRatingsInputPage;
