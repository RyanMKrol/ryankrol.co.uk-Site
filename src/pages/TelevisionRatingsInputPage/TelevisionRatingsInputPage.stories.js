import React from 'react';

import TelevisionRatingsInputPage from './TelevisionRatingsInputPage';

export default {
  title: 'Pages/TelevisionRatingsInputPage',
};

export const Default = () => <TelevisionRatingsInputPage />;
