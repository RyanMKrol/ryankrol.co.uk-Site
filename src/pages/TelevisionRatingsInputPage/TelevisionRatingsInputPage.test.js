import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import TelevisionRatingsInputPage from './TelevisionRatingsInputPage';

it('renders TelevisionRatingsInputPage correctly', () => {
  const header = shallow(<TelevisionRatingsInputPage />);
  expect(toJson(header)).toMatchSnapshot();
});
