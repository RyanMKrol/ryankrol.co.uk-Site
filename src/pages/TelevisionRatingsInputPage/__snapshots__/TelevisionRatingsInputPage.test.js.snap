// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TelevisionRatingsInputPage correctly 1`] = `
<div
  className="container"
>
  <h1
    className="light"
  >
    Rate a TV show!
  </h1>
  <TelevisionRatingsForm />
</div>
`;
